Pomodoro.DataAccess/EF/AppDbContext.cs

```diff
@@ -19,7 +19,10 @@ public class AppDbContext : IdentityDbContext<PomoIdentityUser, IdentityRole<Gui
         public DbSet<TaskEntity> Tasks => Set<TaskEntity>();
         public DbSet<AppUser> AppUsers => Set<AppUser>();
 
-        public AppDbContext(DbContextOptions options) : base(options) { }
+        public AppDbContext(DbContextOptions options) : base(options)
+        {
+            ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
```

Comment:
Why tracking disabled on DbContext level? It's destroyed UnitOfWork pattern implemented in EF.
Maybe better approach use AsNoTracking() for read-only collections retriewing?

Reply:
I fixed it, could you please check the last commit?